### PR TITLE
Removed withVersion:9

### PR DESCRIPTION
**Pull request by jmkiley**
Removed `withVersion:` from the current examples since it is not necessary in v3.6.0.

### CODE REVIEW

File: Examples/ObjectiveC/ExtrusionsExample.m [hunk 1]
```diff
@@ -21,7 +21,7 @@ @implementation ExtrusionsExample
 - (void)viewDidLoad {
     [super viewDidLoad];
     
-    MGLMapView *mapView = [[MGLMapView alloc] initWithFrame:self.view.bounds styleURL:[MGLStyle lightStyleURLWithVersion:9]];
+    MGLMapView *mapView = [[MGLMapView alloc] initWithFrame:self.view.bounds styleURL:[MGLStyle lightStyleURL]];
```
Comment (1ec5) [inline Examples/ObjectiveC/ExtrusionsExample.m]: This example relies on details about the style’s existing source. We should keep specifying the version in such cases.

**Comment by captainbarbosa** [comment]
Awesome @jmkiley, thanks for the heads up. I'm working on a fill pattern example so I'll be sure to follow the new format being used.

**Comment by friedbunny** [comment]
@captainbarbosa if your fill pattern example relies on data-driven styling at all, it should stay with a versioned style URL.

**Comment by jmkiley** [comment]
Awesome! I'm going to go back through tomorrow to check for any other potential source layer related issues. I'll add comments to make reasoning behind the choice more clear.

**Pull request merged**